sdk/identity/azure-identity/azure/identity/_internal/shared_token_cache.py

```diff
@@ -31,22 +31,16 @@
     CacheItem = Mapping[str, str]
 
 
-MULTIPLE_ACCOUNTS = """Multiple users were discovered in the shared token cache. If using DefaultAzureCredential, set
-the AZURE_USERNAME environment variable to the preferred username. Otherwise,
-specify it when constructing SharedTokenCacheCredential.
-Discovered accounts: {}"""
+MULTIPLE_ACCOUNTS = """SharedTokenCacheCredential authentication unavailable. Multiple accounts
```

Comment:
Should have the env var name here?  Seems like we can be a little more helpful than just 'username', 'tenant'  @schaabs

Reply:
`DefaultAzureCredential` will pull values for these from the environment but the credential itself accepts them only as constructor arguments.